src/goto-programs/goto_model.h

```diff
@@ -18,6 +18,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "abstract_goto_model.h"
 #include "goto_functions.h"
 
+#include <iostream>
```

Comment:
`#include <iostream>` in a header is a very bad idea. Please remove this and move the implementation of `check_id_consistency_symbol_table` to a `.cpp` file.

Reply:
Ok, removed `iostream` and reworked to use `messaget` instance.